wrapper:
  config_file_name: urbannav.yaml
  lidar_topic: /velodyne_points
  imu_topic: /imu/data
  lidar_type: 1 # velodyne
  time_unit: 1.0
front_end:
  extrin_r: [1,0,0,
            0,1,0,
            0,0,1]
  extrin_t: [0,0,0.28]
  use_inv: 1
  trajectory_save: true
  trajectory_save_file: m2dgr_street05.txt
ieskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.0001
  cov_bias_gyroscope: 0.0001
